pkg/kubectl/cmd/util/helpers.go

```diff
@@ -834,3 +834,21 @@ func ShouldIncludeUninitialized(cmd *cobra.Command, includeUninitialized bool) b
 	}
 	return shouldIncludeUninitialized
 }
+
+// CheckConflictFlagSettings will check whether the flags can be set explicitly at the same time
+func CheckConflictFlagSettings(cmd *cobra.Command, flagNames ...string) error {
+	if len(flagNames) < 2 {
+		return nil
+	}
+	isCoexist := false
+	for _, flagName := range flagNames {
+		if cmd.Flags().Changed(flagName) {
+			if !isCoexist {
```

Comment:
I think something like this will be easier to read:
```go
if cmd.Flags().Changed(flagName) {
    if isCoexist {
        return fmt.Errorf("cannot set %s at the same time", strings.Join(flagNames, ","))
    }
    isCoexist = true
}
```

Reply:
@soltysh Updated. PTAL. Thanks.